### ISSUE TITLE
return value of a task is lost

### ISSUE THREAD

**Issue by msimonin** [BeyondTheClouds/enoslib#27]
@rcherrueau reported :
+ EnosTask wraps a function `fn` and, at some point executes it without returning it's value.
+ When you use a task as an API, you lose the value returned by `fn`. 
+ See, https://github.com/BeyondTheClouds/enoslib/blob/4e11805c60cbbe8ce18144259da59070fe85de48/enoslib/task.py#L52
+ Can we do something à la FP, `fn` returns a tuple made of the value, plus the updated env? (res, env) ou just res

**Comment by quentinLeDilavrec** [comment]
On my version, I have made the following changes,
``` python
r = fn(*args, **kwargs)
logger.info("- Task %s finished -" % fn.__name__)
return r
```
The use case being that I wanted to be able to loop over multiple inventories from the cli and use the logger of click to see the progress

**Comment by msimonin** [comment]
closing as it has been merged in #41
